amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -14,16 +19,114 @@ def test_authenticateWebAppUser(self):
         raise NotImplementedError()
 
     def test_addAGLogin(self):
-        raise NotImplementedError()
+        # test new user
```

Comment:
Using random indeces to test is best to avoid if you can.

At the very least, you'll want to initialize your random seed `np.random.seed(0)`